corehq/apps/cloudcare/static/cloudcare/js/formplayer/menus/api.js

```diff
@@ -93,7 +93,7 @@ hqDefine("cloudcare/js/formplayer/menus/api", function () {
                                 return;
                             }
 
-                            formplayerUtils.startIntervalSync(params.appId, parsedMenus.metaData.attemptRestore);
+                            formplayerUtils.startIntervalSync(params.appId, parsedMenus?.metaData?.attemptRestore);
```

Comment:
I don't think I've seen this syntax before. I like it (although my IDE doesn't).

Reply:
my IDE doesn't either. I also just learned about it https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Optional_chaining